rocket/rocket.py

```diff
@@ -88,13 +91,19 @@ def launch(
                     modified_files=watcher.modified_files,
                     dbfs_path=dbfs_path,
                     project_location=project_location,
+                    glob_path=glob_path
```

Comment:
This might not be needed as you can filter after the filewatcher looked at all updated files